README.md

```diff
@@ -48,30 +70,51 @@ processes:
     config:
       start_command: '/usr/local/rvm/bin/rvm jruby-9.3.2.0 do bundle exec honeybadger exec traject -c ./lib/traject/config/sirsi_config.rb -s solr_writer.max_skipped=-1 -s log.level=debug -s log.file=log/traject_marc_bodoni_prod_indexer.log'
 ```
+daemon processes run continuously. you can use `eye info` in an ssh session to view status information:
+```sh
+$ eye info
+
+traject
+  workers
+    earthworks-stage-indexer_0 .... up  (12:07, 0%, 73Mb, <1218016>)
+    folio_dev_indexer_0 ........... up  (14:34, 6%, 137Mb, <1386323>)
+    marc_bodoni_dev_indexer_0 ..... up  (12:07, 0%, 2731Mb, <1223573>)
+    marc_morison_dev_indexer_0 .... up  (12:07, 0%, 1253Mb, <1223963>)
+    sw_dev_indexer_0 .............. up  (12:07, 0%, 75Mb, <1224054>)
+    sw_dev_indexer_1 .............. up  (12:07, 0%, 73Mb, <1224303>)
+    sw_preview_stage_indexer_0 .... up  (12:07, 0%, 73Mb, <1224638>)
+    sw_preview_stage_indexer_1 .... up  (12:07, 0%, 73Mb, <1224861>)
+```
+you can stop and start daemons with e.g. `eye stop sw_dev_indexer`. note that it may take some time for all the processes to start and stop. for more information on eye, use `eye help` or see the [eye wiki](https://github.com/kostya/eye/wiki).
 
-As of September 2019, the `config/settings.yml` are not managed in sul-dlss/shared_configs (for better or worse).
-
-
-# Indexing Strategies
+### indexing the data into solr
+traject configurations specific to each target environment are responsible for transforming the data into a format that can be indexed into solr. you can view the configuration files in `lib/traject/config/`, which often include traject commands like:
+```ruby
+to_field 'pub_country', extract_marc('008')
+```
+which extracts information from the 008 field of a MARC record and puts it into the `pub_country` field of the solr document.
 
-## Sirsi
+each traject configuration specifies a reader class located in `lib/traject/readers/` that can read the data from the kafka topic and hand it off to be transformed into solr JSON.
 
+other configuration values, like the URL of the solr instance, are usually set at the top of the configuration file using traject's `provide`. many can be set by environment variables; some of these in turn are set by the eye configuration:
+```yaml
+processes:
+  - name: my_indexer
+    env:
+      SOLR_URL: http://sul-solr.stanford.edu/solr/searchworks-prod # controls where traject indexes to
+```
+in local development, you can invoke traject on the command line and override these settings by passing environment variables:
+```sh
+SOLR_URL=http://localhost:8983/solr/core-name bundle exec traject -c lib/traject/config/sirsi_config.rb my_marc_file.marc
```

Comment:
I found it helpful to know that you can do things like:
`SOLR_URL=http://localhost:8983/solr/core-name bundle exec traject -c lib/traject/config/sirsi_config.rb -w Traject::DebugWriter /some/path/file.ext` to output data for debugging

Also, and this is probably too specific for a general README, but I also debugged using the JSON writer
`SOLR_URL=http://localhost:8983/solr/core-name bundle exec traject -c lib/traject/config/sirsi_config.rb -w Traject::JsonWriter /Users/marloelilongley/Downloads/2306638.marc | tail -n +2 | jq '.toc_struct'` (removes the honeybadger line and then puts the section of output you want into something readable)

Reply:
whoa, i didn't even know you could do this! this definitely belongs in the readme.